backend/JavaWebService/src/main/java/com/kainos/ea/infrastructure/DataBaseRepository.java

```diff
@@ -0,0 +1,29 @@
+package com.kainos.ea.infrastructure;
+
+import org.apache.ibatis.io.Resources;
+import org.apache.ibatis.session.SqlSession;
+import org.apache.ibatis.session.SqlSessionFactory;
+import org.apache.ibatis.session.SqlSessionFactoryBuilder;
+
+import java.io.IOException;
+import java.io.Reader;
+
+public class DataBaseRepository {
+    private SqlSession sqlSession;
+    public DataBaseRepository(SqlSession sqlSession){
+        this.sqlSession = sqlSession;
+    }
+    public void initDBConnection(){
+        try (Reader settings = Resources.getResourceAsReader("mybatis-config.xml")) {
+            SqlSessionFactoryBuilder mybatis = new SqlSessionFactoryBuilder();
+            SqlSessionFactory mappedDb = mybatis.build(settings);
+            sqlSession = mappedDb.openSession();
+        } catch (IOException e) {
+            e.printStackTrace();
```

Comment:
Consider if this is a situation where you _want_ your application to crash. At the moment it will just continue, having printed to the console, possibly crashing later on when you try to use db connectivity.